crates/ethereum/node/src/node.rs

```diff
@@ -32,6 +35,14 @@ use reth_transaction_pool::{
 
 use crate::{EthEngineTypes, EthEvmConfig};
 
+/// Ethereum primitive types.
+#[derive(Debug)]
+pub struct EthPrimitives;
+
+impl NodePrimitives for EthPrimitives {
+    type Block = EthBlock;
+}
```

Comment:
I like this as the next step, can we focus this doing this first and then gradually shift introduce the traits on our primitives (Block/Transactions)

because imo this pr does too much at once.

we need to restrict the NodePrimtives type to `Block = reth_primitives::Block` when we introduce this trait in the components. this way we can start transitioning things without having to do the actual primitive types split (eth, op)